addons/sale_expense/tests/test_reinvoice.py

```diff
@@ -57,7 +57,7 @@ def test_expenses_reinvoice(self):
                     'date': '2016-01-01',
                     'product_id': self.company_data['product_order_sales_price'].id,
                     'unit_amount': self.company_data['product_order_sales_price'].list_price,
-                    'analytic_distribution': {self.analytic_account_1.id: 100},
+                    'analytic_distribution': {self.analytic_account_1.id: 50, self.analytic_account_2.id: 50},
```

Comment:
AFAICS, you're only reusing an existing test here, can't you add a new one to cover this case instead ?

Reply:
Ok, done